__tests__/app.test.js

```diff
@@ -36,12 +36,47 @@ return request(app)
 
 describe('GET /api', () => {
     test('responds with an object describing all the available endpoints on api fetched from the JSON file', () => {
-        console.log(endpoints)
        request(app)
        .get('/api')
        .expect(200)
        .then((response) => {
             expect(response.body).toEqual(endpoints)
        })
     })
+})
+describe('GET /api/articles/:article_id', () => {
+    test('status 200: responds with an article object with relevant properties', () => {
+        return request(app)
+        .get('/api/articles/4')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article).toMatchObject({
+               author: expect.any(String),
+               title: expect.any(String),
+               article_id: expect.any(Number),
+               body: expect.any(String),
+               topic: expect.any(String),
+               created_at: expect.any(String),
+               votes: expect.any(Number),
+               article_img_url: expect.any(String)
+            })
```

Comment:
Since we are looking to get one specific article back, we need to be a lot more specific in our expectations here